website/docs/docs/build/snapshots.md

```diff
@@ -295,6 +444,27 @@ Snapshots can be configured from both your `dbt_project.yml` file and a `config`
 
 Note: BigQuery users can use `target_project` and `target_dataset` as aliases for `target_database` and `target_schema`, respectively.
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+| Config | Description | Required? | Example |
+| ------ | ----------- | --------- | ------- |
+| Config | Description | Required? | Example |
+| ------ | ----------- | --------- | ------- |
+| [database] | The database that dbt should render the snapshot table into | No | analytics |
+| [schema] | The schema that dbt should render the snapshot table into | No | snapshots |
+| [strategy](/reference/resource-configs/strategy) | The snapshot strategy to use. One of `timestamp` or `check` | Yes | timestamp |
```

Comment:
Should we also include `alias`? https://docs.getdbt.com/reference/resource-configs/alias

`database`, `schema`, and `alias` are all "general configs"

Reply:
Good call! I'll add it